src/workflow/ApplyMeasure.cpp

```diff
@@ -276,7 +276,16 @@ end
       } else if (measureType == MeasureType::EnergyPlusMeasure) {
         static_cast<openstudio::measure::EnergyPlusMeasure*>(measurePtr)->run(workspace_.get(), runner, argmap);
       } else if (measureType == MeasureType::ReportingMeasure) {
-        if (energyplus_output_requests) {
+        if (apply_measure_type == ApplyMeasureType::ModelOutputRequests) {
+          if ((*thisEngine)->hasMethod(measureScriptObject, "modelOutputRequests")) {
+            auto n = model.numObjects();
+            LOG(Debug, "Calling measure.modelOutputRequests for '" << measureDirName << "'");
+            static_cast<openstudio::measure::ReportingMeasure*>(measurePtr)->modelOutputRequests(model, runner, argmap);
+            LOG(Debug, "Finished measure.modelOutputRequests for '" << measureDirName << "', " << (model.numObjects() - n) << " objects added");
+          } else {
+            LOG(Debug, "Reporting Measure '" << measureDirName << "' does not have a modelOutputRequests method");
+          }
+        } else if (apply_measure_type == ApplyMeasureType::EnergyPlusOutputRequest) {
```

Comment:
Since I did the work, I'm keeping it to output a debug message

Reply:
oh i see.